connect/runtime/src/test/resources/test-plugins/bad-packaging/META-INF/services/org.apache.kafka.connect.storage.Converter

```diff
@@ -14,6 +14,7 @@
  # limitations under the License.
 
 test.plugins.CoLocatedPlugin
+test.plugins.DefaultConstructorThrowsConverter
 test.plugins.MissingSuperclassConverter
 test.plugins.NoDefaultConstructorConverter
-test.plugins.NoDefaultConstructorConverter
```

Comment:
Just cleaning up (by removing a duplicate plugin declaration) while I was in the neighborhood.